paasta_tools/api/views/instance.py

```diff
@@ -90,6 +90,10 @@
 log = logging.getLogger(__name__)
 
 
+INSTANCE_TYPES_K8S = {"flink", "cassandracluster"}
```

Comment:
there is `INSTANCE_TYPES` in `paasta_tools/utils.py`, may be move `INSTANCE_TYPES_K8S ` there?

Reply:
will do, thanks!